### PR TITLE
[FIX] scrubbing parameter names in `nilearn.input_data.fmriprep_confounds` not matching docstring

### PR DESCRIPTION
**Pull request by htwangtw**
In `nilearn.input_data.fmriprep_confounds`, two parameter names are not matching the docstrings.

`fd_thresh` -> `fd_threshold`
`std_dvars_thresh` -> `std_dvars_threshold`

### CODE REVIEW

File: nilearn/input_data/fmriprep_confounds_strategy.py [hunk 1]
```diff
@@ -145,22 +145,22 @@ def fmriprep_confounds_strategy(img_files, denoise_strategy="simple",
...
-        N/A              N/A            N/A       N/A       True*
-        scrubbing True      full*  full   None*         5*    0.2*      \
-        3*               N/A            N/A       N/A       True*
-        compcor   True      full*  N/A    N/A           N/A   N/A       \
-        N/A              anat_combined* all*      N/A       True*
-        ica_aroma True      N/A    basic* None*         N/A   N/A       \
-        N/A              N/A            N/A       full      True*
-        ========= ========= ====== ====== ============= ===== ========= \
-        ================ ============== ========= ========= ======
+        ========= ========= ====== ====== ============= ===== ============ \
```
Comment (NicolasGensollen) [inline nilearn/input_data/fmriprep_confounds_strategy.py:148]: It looks like the rendering of the table is not working anymore.

**Review by NicolasGensollen — COMMENTED** [review]
Thanks for spotting and fixing this @htwangtw !
LGTM minus the rendering of the table.

File: nilearn/input_data/fmriprep_confounds_strategy.py (hunk 1 above)
Comment (htwangtw) [reply nilearn/input_data/fmriprep_confounds_strategy.py:148]: Now it's fixed!

**Review by NicolasGensollen — APPROVED** [review]
LGTM, thanks @htwangtw !

**Review by bthirion — APPROVED** [review]
LGTM, thx.

**Comment by NicolasGensollen** [comment]
Thanks!
Merging...

**Pull request merged**